app/processing/genesis/transform-invoice-line.js

```diff
@@ -0,0 +1,23 @@
+const transformInvoiceLine = (lineData) => {
+  return [{
+    companyCode: lineData[2],
```

Comment:
is it worth a comment or updating the docs somewhere to explain why there are two invoice lines here as it contrast with the other schemes?

Reply:
Added a comment to explain the reasoning